### ISSUE TITLE
Improvements in player notifications

### ISSUE THREAD

**Issue by NeeRaj-2401** [libre-tube/LibreTube#2722]
### Describe your suggested feature

After #2713 player notifications has been improved alot. But there is few things i noticed that should be improved

[Bug]
1. Video thumbnail disappears from notification as soon as we leave the app in background and either forward/backward it from notification.

[Feature]
2. Organize the notification media controls in order
    Previous - backward - play/pause - forward - next (looks good imo)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by xIUPITERx** [comment]
> Previous - backward - play/pause - forward - next (looks good imo)

You mean instead of _backward - Previous - play/pause - next - forward_

**Comment by RickyM7** [comment]
Another thing that's missing is a button to stop the rreproduction and close the notification. There's not a way to close the notification when the queue ends for now, and it's a problem.

**Comment by Bnyro** [comment]
1. is fixed
2. is not planned because that would overcomplicate things with the notification on Android 13+
